### PR TITLE
Add qwen speculative example

### PR DESCRIPTION
**Pull request by hzjane**
## Description
Add qwen speculative example

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by jason-dai** [comment]
Please update readme similar to https://github.com/intel-analytics/BigDL/pull/9986

File: python/llm/example/CPU/Speculative-Decoding/qwen/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Qwen
+In this directory, you will find examples on how you could run Qwen BF16 infernece with 
+self-speculative decoding using BigDL-LLM on Intel CPUs. For illustration purposes, we utilize the [Qwen/Qwen-7B-Chat](https://huggingface.co/Qwen/Qwen-7B-Chat) and [Qwen/Qwen-14B-Chat](https://huggingface.co/Qwen/Qwen-14B-Chat) as reference Qwen models.
```
Comment (glorysdj) [inline python/llm/example/CPU/Speculative-Decoding/qwen/README.md:3]: also add Qwen 72B

File: python/llm/example/CPU/Speculative-Decoding/qwen/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+### 3. Run
+
+```
+(numactl -m 0 -C 0-47) python ./speculative.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH 
+--prompt PROMPT --n-predict N_PREDICT
+```
+
+Arguments info:
+- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the Qwen model (e.g. `Qwen/Qwen-7B-Chat` and `Qwen/Qwen-14B-Chat`) to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'Qwen/Qwen-7B-Chat'`.
```
Comment (glorysdj) [inline python/llm/example/CPU/Speculative-Decoding/qwen/README.md:30]: deafult to Qwen-14B?

File: python/llm/example/CPU/Speculative-Decoding/qwen/README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+```
+(numactl -m 0 -C 0-47) python ./speculative.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH 
+--prompt PROMPT --n-predict N_PREDICT
+```
+
+Arguments info:
+- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the Qwen model (e.g. `Qwen/Qwen-7B-Chat` and `Qwen/Qwen-14B-Chat` and `Qwen/Qwen-72B-Chat`) to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'Qwen/Qwen-14B-Chat'`.
+- `--prompt PROMPT`: argument defining the prompt to be infered (with integrated prompt format for chat). A default prompt is provided.
+- `--n-predict N_PREDICT`: argument defining the max number of tokens to predict. It is default to be `128`.
```
Comment (glorysdj) [inline python/llm/example/CPU/Speculative-Decoding/qwen/README.md:32]: It is set to `128` by default.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**